docs/developing.rst

```diff
@@ -571,6 +571,24 @@ Example::
 
             toil.exportFile(outputFileID, 'file:///some/other/local/path')
 
+
+There are some trade-offs to file staging that should be considered. It may be more efficient to
+directly download a file for cases in which the cost of the copy operation imposed by an import is
+unnecessary. For example, a direct download is better for large files that are used only once.
+Importing can be more efficient in cases where the file in question is needed several times. For
+example, Amazon's S3 platform, used in the AWS job store, is better optimized for reads and writes
+to files that are in the same region as that of the job store. Thus an import is justified if a
```

Comment:
Actually, it is optimized for access by EC2 instances running in the same region as the bucket.